### PR TITLE
Remove unnecessary semicolons in source code

### LINKED ISSUE #130: Remove semicolon at the ends of if statements
Some `if `statements have unnecessary semicolon at the end.
Although the code can still compile, the unnecessary semicolon should be removed.

### PR DESCRIPTION
**Pull request by chihtaw**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ SPDX-License-Identifier: GPL-3.0-or-later
 -->
 
 # 0.5.4 (unreleased)
+- Remove unnecessary semicolon in source code [PR #131] (https://github.com/hpsim/OGL/pull/131)
```
Comment (greole) [inline CHANGELOG.md]: Not needed to add to the changelog since it has no effect for the user.
```suggestion
```

Comment (chihtaw) [reply CHANGELOG.md]: Okay, but it will fail the corresponding check.

Comment (greole) [reply CHANGELOG.md]: No worries, we can still merge. It is not a hard requirement. We could also add an optional flag for the changelog test.

**Comment by chihtaw** [comment]
@greole Do you think it is okay to merge?

**Review by greole — APPROVED** [review]

**Pull request merged**